cloud/azure/bin/setup-keyvault

```diff
@@ -44,4 +42,5 @@ echo "Generating and setting secrets"
 key_vault::add_generated_secrets \
   "${vault_name}" \
   "${POSTGRES_PASSWORD_NAME}" \
-  "${APP_SECRET_KEY_NAME}"
+  "${APP_SECRET_KEY_NAME}" \
+  "${ADFS_SECRET_NAME}"
```

Comment:
instead of using the generated secret store a fake secret that is obvious for debugging purposes for the placeholders that we make